protocol/storage/miner_test.go

```diff
@@ -66,31 +54,26 @@ func TestReceiveStorageProposal(t *testing.T) {
 		_, err := miner.receiveStorageProposal(context.Background(), proposal)
 		require.NoError(t, err)
 
-		assert.True(t, accepted, "Proposal has been accepted")
-		assert.False(t, rejected, "Proposal has not been rejected")
-		assert.Equal(t, "", message)
+		// one deal should be stored and it should have been accepted
```

Comment:
add 'and signed'  since this is being checked here

Reply:
I broke out the assertion that the response is signed when the deal is accepted into a separate test.